### ISSUE TITLE
Ipad Screen Support

### ISSUE THREAD

**Issue by vanderLo** [rileytestut/AltStore#39]
Large screen support would be nice, but not essential :)

**Comment by noah978** [comment]
Full iPad support is planned for the future. It might take a little extra time since full iPad support also assumes multi-device support and there are a lot of factors in play with that.

**Comment by isametry** [comment]
Does this include [Delta](https://github.com/rileytestut/Delta)?

**Comment by noah978** [comment]
Yes of course iPad support for Delta is planned as well. It will probably be coming *before* AltStore has full iPad support as well.

**Comment by th3sn3k** [comment]
> Large screen support would be nice, but not essential :)

There is a workaround for now if you’re jailbroken install nomoresmallapps and landscape works occupying the entire screen.

**Comment by FabiRich** [comment]
Are there any updates on that matter, as side loading on iPad OS has been possible in the EU for over a year https://developer.apple.com/de/support/dma-and-apps-in-the-eu/
